package.json

```diff
@@ -28,17 +29,17 @@
     "url": "git+https://github.com/vector-im/compound-design-tokens.git"
   },
   "devDependencies": {
-    "@tokens-studio/sd-transforms": "^0.4.3",
-    "@types/fs-extra": "^11.0.1",
-    "@types/lodash": "^4.14.191",
-    "@types/node": "^18.13.0",
-    "fast-glob": "^3.2.12",
+    "@tokens-studio/sd-transforms": "^0.12.2",
+    "@types/fs-extra": "^11.0.4",
+    "@types/lodash-es": "^4.17.12",
+    "@types/node": "^20.14.6",
+    "fast-glob": "^3.3.2",
     "http-server": "^14.1.1",
-    "style-dictionary": "https://github.com/amzn/style-dictionary",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.9.5"
+    "lodash-es": "^4.17.21",
+    "style-dictionary": "3.8.0",
+    "svg2vectordrawable": "^2.9.1",
+    "tsx": "^4.15.6",
+    "typescript": "^5.4.5"
   },
-  "dependencies": {
-    "svg2vectordrawable": "^2.9.1"
-  }
```

Comment:
I was always a little bit annoyed that this was in my tree for some reason in MAS…